core/src/main/scala/org/apache/spark/scheduler/LiveListenerBus.scala

```diff
@@ -39,20 +41,13 @@ import org.apache.spark.util.Utils
  * has started will events be actually propagated to all attached listeners. This listener bus
  * is stopped when `stop()` is called, and it will drop further events after stopping.
  */
-private[spark] class LiveListenerBus(conf: SparkConf) extends SparkListenerBus {
-
-  self =>
+private[spark] class LiveListenerBus(conf: SparkConf) {
```

Comment:
is it still valid to call `LiveListenerBus.postToAll`? If not, shall we overwrite `postToAll` and throw exception?

Reply:
also `LiveListenerBus.addListener` seems not valid too.

Reply:
`LiveListenerBus.postToAll` doesn't exist anymore, and neither does `LiveListenerBus.addListener`. But I'm not sure what you're asking.

Reply:
oh sorry I missed that, `LiveListenerBus` doesn't extend `SparkListenerBus` anymore.